using_images/db_images/mysql.adoc

```diff
@@ -172,3 +172,356 @@ The MySQL templates should have been registered in the `openshift` project by th
+    "name":"mysql-master"
+  },
+  "spec":{
+    "strategy":{
+      "type":"Recreate"
+    },
+    "triggers":[
+      {
+        "type":"ConfigChange"
+      }
+    ],
+    "replicas":1,
+    "selector":{
+      "name":"mysql-master"
+    },
+    "template":{
+      "metadata":{
+        "labels":{
+          "name":"mysql-master"
+        }
+      },
+      "spec":{
+        "volumes":[
+          {
+            "name":"mysql-master-data",
+            "persistentVolumeClaim":{
+              "claimName":"mysql-master"
+            }
+          }
+        ],
+        "containers":[
+          {
+            "name":"server",
+            "image":"openshift/mysql-55-centos7",
+            "command":[
+              "run-mysqld-master.sh"
+            ],
+            "ports":[
+              {
+                "containerPort":3306,
+                "protocol":"TCP"
+              }
+            ],
+            "env":[
+              {
+                "name":"MYSQL_MASTER_USER",
+                "value":"${MYSQL_MASTER_USER}"
+              },
+              {
+                "name":"MYSQL_MASTER_PASSWORD",
+                "value":"${MYSQL_MASTER_PASSWORD}"
+              },
+              {
+                "name":"MYSQL_USER",
+                "value":"${MYSQL_USER}"
+              },
+              {
+                "name":"MYSQL_PASSWORD",
+                "value":"${MYSQL_PASSWORD}"
+              },
+              {
+                "name":"MYSQL_DATABASE",
+                "value":"${MYSQL_DATABASE}"
+              },
+              {
+                "name":"MYSQL_ROOT_PASSWORD",
+                "value":"${MYSQL_ROOT_PASSWORD}"
+              }
+            ],
+            "volumeMounts":[
+              {
+                "name":"mysql-master-data",
+                "mountPath":"/var/lib/mysql/data"
+              }
+            ],
+            "resources":{
+
+            },
+            "terminationMessagePath":"/dev/termination-log",
+            "imagePullPolicy":"IfNotPresent",
+            "securityContext":{
+              "capabilities":{
+
+              },
+              "privileged":false
+            }
+          }
+        ],
+        "restartPolicy":"Always",
+        "dnsPolicy":"ClusterFirst"
+      }
+    }
+  }
+}
+----
+====
+
+Since we claimed a persistent volume in this deployment configuration to have
+all data persisted for the MySQL master server, you have to ask administrator
+to create a link:../../admin_guide/persistent_storage_nfs.html#provisioning[persistent volume]
+that you can claim the storage from.
+
+After the deployment configuration is created and the pod with MySQL master server
+is started, it will create the database defined by `*MYSQL_DATABASE*` and configure
+the server to replicate this database to slaves.
+
+Note that the provided example does not support multi-master setup, so you can not
+scale the replication controller created by this deployment config up to one replica.
```

Comment:
> so you can not scale the replication controller created by this deployment config up to one replica

that reads strangely..I think you are trying to say you can't have more than one instance, right?  so really you can't scale it beyond 1 replica.  (which is itself confusing since 1 replica implies there is also 1 primary somewhere, but k8s does call it numReplicas and setting numReplicas to 1 gets you exactly one instance)